lib/features/sks_people_counter_button/repository/latest_sks_user_data_repo.dart

```diff
@@ -0,0 +1,14 @@
+import "package:flutter_riverpod/flutter_riverpod.dart";
+import "package:riverpod_annotation/riverpod_annotation.dart";
+
+import "../api_client/sks_client.dart";
+import "../models/sks_user_data.dart";
+
+part "latest_sks_user_data_repo.g.dart";
+
+@riverpod
+Future<SksUserData> getLatestSksUserData(Ref ref) async {
+  final dio = ref.read(sksClientProvider);
+  final response = await dio.get(SksClientConfig.latestDataEndpoint);
+  return SksUserData.fromJson(response.data as Map<String, dynamic>);
+}
```

Comment:
for the sake of this task this will be enough, but as a next task (or in this one) we need to add interval refreshing.

Actually I've forced @czaja307 to make a few features for us, so in the response you have 2 fields: `nextUpdateTimestamp` and `isResultRecent`.

The logic we want is:
- if `isResultRecent` == true; then we want to refresh after `nextUpdateTimestamp` passes
- if == false; let's check in a minute (we'll need to see if this behaves as expected but that's the theory at least)

Reply:
but you can delegate it to next task if you prefer

Reply:
You can delegate it to next task because I dont have time for next 2 days

Reply:
sure